modules/SofaGraphComponent/SofaGraphComponent_test/SceneChecker_test.cpp

```diff
@@ -82,13 +82,20 @@ struct SceneChecker_test : public BaseSimulationTest
         sofa::component::initSofaBaseUtils();
         sofa::component::initSofaBaseMechanics();
         sofa::component::initSofaMeshCollision();
+
+        sofa::helper::system::PluginManager::getInstance().getPluginMap();
+        auto& map = PluginManager::getInstance().getPluginMap();
+        for (const auto& elem : map)
+        {
+            std::cout <<  elem.second.getModuleName() << std::endl;
+        }
```

Comment:
Is this important? You just `cout` something. Or is there side effects?

Reply:
well spotted, it was for debug purpose and i did not remove this.. 👍